docs/use-cases.rst

```diff
@@ -12,11 +12,100 @@ Typical Use-Cases
 
 .. START SUBSEC:INITIAL-PEERS
 
+.. _use-case-fast-rtps-over-wifi:
+
+Fast-RTPS over WIFI
+===================
+
+The `RTPS standard <https://www.omg.org/spec/DDSI-RTPS/2.2/PDF>`_ defines the SIMPLE discovery as the default
+mechanism for discovering participants in the network.
+One of the main features of this mechanism is the use of multicast communication in the PDP (Participant Discovery
+Phase). This could be a problem in case the communication is not wired, i.e. WiFi communication is used. The solution
+to this challenge is to define the participants with which a unicast communication is to be set. This is the initial
+peers list.
```

Comment:
Phase). This could be a problem in case the communication is not wired, i.e. WiFi communication. Fast-RTPS' solution to this challenge is to define the participants with which a unicast communication is to be set, i.e an initial list of remote peers.